includes/Traits/Find_Readme.php

```diff
@@ -27,12 +27,15 @@ protected function filter_files_for_readme( array $files, $plugin_relative_path
 		// Find the readme file.
 		$readme_list = preg_grep( '/readme\.(txt|md)$/i', $files );
 
+		$plugin_relative_path = rtrim( $plugin_relative_path, '/' );
```

Comment:
Why is this needed? In which situation would `$plugin_relative_path` end in a forward slash?